doc/api/http2.md

```diff
@@ -1977,6 +2053,8 @@ changes:
     remote peer upon connection.
   * ...: Any [`tls.createServer()`][] options can be provided. For
     servers, the identity options (`pfx` or `key`/`cert`) are usually required.
+  * `origins` {string[]} An array of origin strings to send within an `ORIGIN`
```

Comment:
Should this also added to the `http2.createServer()` as per the note in the `serverhttp2session.origin(...origins)` section?

Reply:
Should this be added in YAML changes?

Reply:
No, the note should be updated. The `ORIGIN` frame is only meaningful on https connection.

Reply:
And yes, it likely should be in YAML changes